File: resourcespoet/src/test/java/com/commit451/resourcespoet/StringArrayTest.java
package com.commit451.resourcespoet;

import org.junit.Test;

import java.util.ArrayList;

/**
 * Tests the resources creation
 */
public class StringArrayTest {

    @Test
    public void stringArrayTest() throws Exception {
        ArrayList<String> strings = new ArrayList<>();
        strings.add("One");
        strings.add("Two");

        ResourcesPoet poet = ResourcesPoet.create()
                .addStringArray("stuff", strings);

        TestUtil.assertEquals("string_array.xml", poet);
    }
}

File: resourcespoet/src/test/java/com/commit451/resourcespoet/StringArrayTest.kt
package com.commit451.resourcespoet

import org.junit.Test
import java.util.*

/**
 * Tests the resources creation
 */
class StringArrayTest {

    @Test
    fun stringArrayTest() {
        val strings = ArrayList<String>()
        strings.add("One")
        strings.add("Two")

        val poet = ResourcesPoet.create()
                .addStringArray("stuff", strings)

        TestUtil.assertEquals("string_array.xml", poet)
    }
}
